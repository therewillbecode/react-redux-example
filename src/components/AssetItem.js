import React, { Component } from "react";

const AssetItem = ({ asset }) => (
  <div>
    <h3>{asset.id}</h3>
    {asset.name}
  </div>
);

export default AssetItem;
