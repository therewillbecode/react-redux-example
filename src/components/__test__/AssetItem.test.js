/* eslint-disable */
import React from "react";
import ReactShallowRenderer from "react-test-renderer/shallow";

import AssetItem from "../AssetItem";

describe("AssetItem", () => {
  it("renders correctly", () => {
    const renderer = new ReactShallowRenderer();
    renderer.render(<AssetItem />);
    const tree = renderer.getRenderOutput();
    expect(tree).toMatchSnapshot();
  });
});
