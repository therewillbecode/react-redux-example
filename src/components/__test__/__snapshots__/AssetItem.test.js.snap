// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AssetItem renders correctly 1`] = `
<Connect(EditAssetFormContainer)
  asset={undefined}
/>
`;
